src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -0,0 +1,194 @@
+<template>
```

Comment:
It is more traditonal for css ids and classes to be kebab case. e.g. barcode-input-group